drs/drs-assembly/subscribe-model.conf

```diff
@@ -460,120 +460,198 @@ Current position of the cold stop. Contains the following:
+  name = detectorInfo
+  requiredRate = 1
+  usage = """
+  Information of detector used in IFS. Contains the following:
+  
+  *IFS detector name. FITS=DETNAME
+  *IFS detector read noise. FITS=RNOISE
+  *IFS detector gain. FITS=GAIN
+  
+  """
 }
 
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly
-  name =  RNOISE
-  requiredRate =  1
-  usage = "IFS detector read noise. FITS=RNOISE"
-}
+#{
+#  subsystem =  IRIS
+#  component =  ifs-detector-assembly
+#  name =  DETNAME
+#  requiredRate =  1
+#  usage = "IFS detector name. FITS=DETNAME"
+#}
+#
+#{
+#  subsystem =  IRIS
+#  component =  ifs-detector-assembly
+#  name =  RNOISE
+#  requiredRate =  1
+#  usage = "IFS detector read noise. FITS=RNOISE"
+#}
+#
+#{
+#  subsystem =  IRIS
+#  component =  ifs-detector-assembly
+#  name =  gain
+#  requiredRate =  1
+#  usage = "IFS detector gain. FITS=GAIN"
+#}
 
 {
   subsystem =  IRIS
   component =  ifs-detector-assembly
-  name =  gain
-  requiredRate =  1
-  usage = "IFS detector gain. FITS=GAIN"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  EXPTIME
-  requiredRate =  1
-  usage = "IFS total integration time. FITS=EXPTIME"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  RAMPNUM
-  requiredRate =  4
-  usage = "IFS read number. FITS=RAMPNUM"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  READNUM
-  requiredRate =  4
-  usage = "IFS read number. FITS=READNUM"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  READTIME
-  requiredRate =  4
-  usage = "IFS time in seconds to readout detector. FITS=READTIME"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  groups
-  requiredRate =  4
-  usage = "Total number of IFS ramps in this exposure. FITS=RAMPS"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  intergrationTime
-  requiredRate =  1
-  usage = "IFS integration time of each readout. FITS=ITIME"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  reads
-  requiredRate =  4
-  usage = "Total number of IFS readouts in this ramp. FITS=READS"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  subregionEndX
-  requiredRate =  1
-  usage = "IFS subregion ending x-pos. FITS=SUBREGX1"
-}
-
-{
-  subsystem =  IRIS
-  component =  ifs-detector-assembly.currentConfig
-  name =  subregionEndY
+  name =  currentConfig
   requiredRate =  1
-  usage = "IFS subregion ending y-pos. FITS=SUBREGY1"
+  usage = """
+  Current configuration of the IFS detector. Contains the following:
+  
+  *IFS total integration time. FITS=EXPTIME
+  *IFS read number. FITS=RAMPNUM
+  *IFS read number. FITS=READNUM
+  *IFS time in seconds to readout detector. FITS=READTIME
+  *Total number of IFS ramps in this exposure. FITS=RAMPS
```

Comment:
We decided to pull COADDS from the IS.  RAMPS is trying to keep track of the number of ramps in an observing sequence.  We're also trying to keep the terminology similar to the Imager as well...